SpatialGDK/Documentation/content/get-started/example-project/exampleproject-cloud-deployment.md

```diff
@@ -35,190 +37,274 @@ When you signed up for SpatialOS, your account was automatically associated with
+
+<br/>
+**Build workers manually using the command line - with optional flags** 
+
+Run the `BuildProject.bat` script with added flags from the command line when you want to build your server-workers and client-workers separately,  or, if you want to build different worker configurations. 
+
+<%(#Expandable title="What flags are there?")%>
+
+During development, you might want to, for example:
+
+* cook a headless standalone version of the game ready for upload to the SpatialOS cloud as a simulated player cloud deployment, or
+* cook a stand-alone version of the game to test it as a game client.
+The optional flags give you this functionality and more.
+
+See the [Helper script]({{urlRoot}}/content/apis-and-helper-scripts/helper-scripts) documentation for details of all the options and how to use them.
+
 <%(/Expandable)%>
-For now, you need to build server-workers and client-workers, so if you haven't run `BuildProject.bat` from File Explorer you need to:</br></br>
-    1. Close your Unreal Editor - if the Editor is open when you try to build workers, the command fails.
-    1. Open a terminal window and navigate to the `UnrealGDKExampleProject` directory.
-    1. Run the `BuildProject.bat` command to build a server-worker using the filepath and flags below. </br>
-    The filepath you use depends on whether you have the `UnrealGDK` plugin set up as an *engine* plugin or as a *project* plugin. If you followed the default setup instructions which use the `InstallGDK.bat` script, you have it set up as an *engine* plugin. <br/></br>
-        * Engine plugin filepath (default):</br>
-        ```
-        UnrealEngine\Engine\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat GDKShooterServer Linux Development GDKShooter.uproject
-        ```
-        </br>
-        * Project plugin filepath:</br>
-        ```
-        UnrealGDKExampleProject\Game\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat GDKShooterServer Linux Development GDKShooter.uproject
-        ```
-        </br></br>
-    1. Now run the `BuildProject.bat` command to build a client-worker: <br/><br/>
-        * Engine plugin filepath (default):</br>
-        ```
-        UnrealEngine\Engine\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat GDKShooter Win64 Development GDKShooter.uproject
-        ```
-        <br/>
-        * Project plugin filepath:</br>
-        ```
-         UnrealGDKExampleProject\Game\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat GDKShooter Win64 Development GDKShooter.uproject
-        ```
-        <br/><br/>
+
+For now, you need to build server-workers and client-workers, so if you haven't run `BuildProject.bat` from File Explorer you need to:
```

Comment:
If you haven't yet run `BuildProject.bat` to build server-workers and client-workers, you need to:
* navigate to the `BuildProject.bat` file using File Explorer
* blah blah